modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -763,10 +763,52 @@ class CV_EXPORTS_W QRCodeEncoder {
 
 };
 
+
+struct CV_EXPORTS_W_SIMPLE QRDetectorParameters {
+
+    /** @brief Methods for detecting QR codes: BASE, ARUCO_BASED */
+    enum Method {
+        BASE, ///< old QR detect method
```

Comment:
I propose to name it `HOUGH_BASED` or some other way to highlight the internal approach.